AppCenter/AppCenterTests/Util/MSMockUserDefaults.h

```diff
@@ -9,5 +9,5 @@
  * Clear dictionary
  */
 - (void)stopMocking;
-
+- (void)migrateKeys:( NSDictionary *)migratedKeys forService:( NSString *)serviceName;
```

Comment:
Please run formatter for this change.
Also add a doc for the method.
```suggestion

- (void)migrateKeys:(NSDictionary *)migratedKeys forService:(NSString *)serviceName;

```